lib/debug/session.rb

```diff
@@ -253,7 +253,8 @@ def process_event evt
         else
           tc << [:eval, :display, @displays]
         end
-
+      when :show
+        tc << [:show, :list, {dir: -1}]
```

Comment:
who use it?

Reply:
sorry I'm not sure where this line is. I can't find it in the latest changes.

Reply:
this is now marked as outdated. I guess GH displayed some stale data when you left the comment? I think it's from an earlier implementation.